### ISSUE TITLE
Tardis conversion wrong schema

### ISSUE THREAD

**Issue by volemont** [nkaz001/hftbacktest#160]
I got the following error while converting Tardis data:
```
In [18]: 
Reading binance-futures_incremental_book_L2_YFIUSDT_20230523.csv.gz
could not parse `56.7` as dtype `i64` at column 'price' (column number 7)

The current offset in the file is 134208 bytes.

You might want to try:
- increasing `infer_schema_length` (e.g. `infer_schema_length=10000`),
- specifying correct dtype with the `dtypes` argument
- setting `ignore_errors` to `True`,
- adding `56.7` to the `null_values` list.

Original error: ```remaining bytes non-empty
```
I defined schemas in https://github.com/nkaz001/hftbacktest/blob/415f81b3c910572ce8de0934b24e73a52daf9884/py-hftbacktest/hftbacktest/data/utils/tardis.py#L85-L87 to work around the issue:
```
        if "trades" in file:
            schema = {
                'exchange': pl.String,
                'symbol': pl.String,
                'timestamp': pl.Int64,
                'local_timestamp': pl.Int64,
                'id': pl.UInt64,
                'side': pl.String,
                'price': pl.Float64,
                'amount': pl.Float64,
            }
        elif "incremental_book_L2" in file:
            schema = {
                'exchange': pl.String,
                'symbol': pl.String,
                'timestamp': pl.Int64,
                'local_timestamp': pl.Int64,
                'is_snapshot': pl.Boolean,
                'side': pl.String,
                'price': pl.Float64,
                'amount': pl.Float64,
            }
        else:
            raise ValueError(f"Unknown file type: {file}")
        print('Reading %s' % file)
        df = pl.read_csv(file, schema=schema)
```
Maybe there is a more elegant way to solve the problem.

**Comment by AdityaRoot** [comment]
Similar error as the above, although I'm unfortunately not skilled enough to know how to fix it so would appreciate any guidance. For tarids using trades (currently using bybit BEERUSDT June 1st 2024 data)


```
ComputeError: could not parse `dfc57692-1b1b-5816-bda5-742c338b0db5` as dtype `u64` at column 'id' (column number 5)

The current offset in the file is 53 bytes.

You might want to try:
- increasing `infer_schema_length` (e.g. `infer_schema_length=10000`),
- specifying correct dtype with the `schema_overrides` argument
- setting `ignore_errors` to `True`,
- adding `dfc57692-1b1b-5816-bda5-742c338b0db5` to the `null_values` list.

Original error: ```remaining bytes non-empty```
```

And receiving the above when trying to import the trades file. I assume that the error is simply that the length is incorrect for the id column?

**Comment by nkaz001** [comment]
Which was fixed in https://github.com/nkaz001/hftbacktest/commit/6e867df4022d3ac5c13223a7c590ed28714c4dfe .

**Comment by AdityaRoot** [comment]
> Which was fixed in [6e867df](https://github.com/nkaz001/hftbacktest/commit/6e867df4022d3ac5c13223a7c590ed28714c4dfe) .

Ah okay, thank you! I'll implement the override manually into the library :)
